boards/samd20_xplained_pro/SAMD20.yaml

```diff
@@ -0,0 +1,1329 @@
+---
```

Comment:
what is this file for?

Reply:
This is the chip description file to include when using cargo flash using `--chip-description-path` argument